.github/workflows/ci.yml

```diff
@@ -66,9 +66,11 @@ jobs:
         run: |
           pytest -v --cov=cinnabar --cov-report=xml --cov-report=term --color=yes cinnabar/tests/
 
-      - uses: codecov/codecov-action@v3
+      - name: codecov
+        if: ${{ github.repository == 'OpenFreeEnergy/cinnabar' && github.event_name != 'schedule'}}
+        uses: codecov/codecov-action@v3
         with:
           token: ${{ secrets.CODECOV_TOKEN }}
           files: ./coverage.xml
-          fail_ci_if_error: true # optional (default = false)
+          fail_ci_if_error: false
```

Comment:
you're always best to fail here, that way you know to restart the build to get coverage information, otherwise you'll have misleading info at best

Reply:
codecov is flakey and advisory, I'd rather it not fail a PR

Reply:
it really doesn't fail that much tbh - I've seen conda resolution errors more often

Plus on runners of this length, it's no big deal to just restart it or just check that CI ran and then ignore the fail

Reply:
it is a lot worse if you ask someone to add test coverage when it's already covered